src/components/styles/layout.module.css

```diff
@@ -0,0 +1,33 @@
+.layoutBox {
+    margin: 3em auto;
+    margin-top: 0;
+    max-width: 650;
+    padding: 0;
+}
+
+.navBox {
+    text-align: center;
+    background: black;
+    border: black;
+    padding: .5em;
+}
+
+.navLink {
+    color: blanchedalmond;
+    margin: 0 5vw;
+    box-shadow: none;
+    text-decoration: none;
+}
+
+.title {
+    display: inline-block;
+    color: blanchedalmond;
+    margin: 0 30vw 0 0;
+    box-shadow: none;
+    text-decoration: none;
+}
+
+
+/* div {
+    border: 1px solid blue;
+} */
```

Comment:
This probably isn't relevant for a solo blog project, but usually you want to avoid committing commented-out code like this — it has a tendency to hang around forever, and other people trying to maintain your work may be confused as to why it's there in the first place.